cfgov/jinja2/v1/_includes/organisms/main-contact-info.html

```diff
@@ -39,52 +39,38 @@
 {% import 'molecules/contact-email.html' as contact_email %}
 {% import 'molecules/contact-phone.html' as contact_phone %}
 {% import 'molecules/contact-address.html' as contact_address %}
-{% if value %}
-    {% if value.contact %}
-        <div class="o-main-contact-info">
-            <h2>{{ value.contact.heading }}</h2>
 
```

Comment:
wrong section was removed. we want to get rid of the macro and leave the `value` related code intact. if the macro is being used else where then leave template as is and make a TODO note.